cmd/utils.go

```diff
@@ -272,15 +273,54 @@ func getProjectConfig(config *RootCommandConfig) (ProjectConfig, error) {
 
 		}
 		appsodyConfig := filepath.Join(dir, ConfigFile)
+
 		v := viper.New()
 		v.SetConfigFile(appsodyConfig)
 		Debug.log("Project config file set to: ", appsodyConfig)
+
 		err := v.ReadInConfig()
+
 		if err != nil {
 			var tempProjectConfig ProjectConfig
 			return tempProjectConfig, errors.Errorf("Error reading project config %v", err)
 
 		}
+
+		if prjName != "" {
+			prjName = strings.ToLower(prjName)
```

Comment:
The logic should be as we discussed in the issue: if the user provides a string, we should validate it against the Kubernetes regex. If that validation fails, we should simply tell the user that the name they provided can't be used. We shouldn't manipulate the string that is given to us.

Reply:
I have changed it; however, I was manipulating the string because I was thinking it was better to manipulate the string and the let user know the name of the project rather than displaying the error.